x-pack/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -89,6 +90,7 @@ export class TabNavigation extends React.PureComponent<TabNavigationProps, TabNa
       ...this.state,
       selectedTabId: id,
     });
+    trackUiAction('tab_' + id);
```

Comment:
For here, I would use interpolation rather than the concat + syntax. That plus my other comment and this would be:

```ts
track(`tab_${id}`)
```